src/api/database/bulk-api.rst

```diff
@@ -322,10 +322,10 @@ documents created, here with the combination and their revision IDs:
         }
     ]
 
-The content and structure of the returned JSON will depend on the transaction
-semantics being used for the bulk update; see :ref:`api/db/bulk_docs/semantics`
-for more information. Conflicts and validation errors when updating documents
-in bulk must be handled separately; see :ref:`api/db/bulk_docs/validation`.
+For details of the schematic content and structure of the returned JSON see
```

Comment:
schematic -> semantic?

That is the "meaning" of various parts of the response

Reply:
ha... thanks for catching my typo.